src/app/logic/ComputerPlayer.js

```diff
@@ -1,18 +1,23 @@
 import getRandomInt from '../utils/getRandomInt';
+import { Player } from './Player';
+import isAnswerCorrect from './isAnswerCorrect';
 
-export const ComputerPlayer = () => {
-  const player = {
-    askQuestion(question, callback) {
-      if (!question) throw new Error('There is no question');
-      callback(question);
-    },
+class ComputerPlayer extends Player {
```

Comment:
Consider add constructor with call to Player constructor

Reply:
Could please explain for what? ComputerPlayer doesn't need it's own constructor, I was sure that when a class extends another class and has no constructor, then there is generated 'empty' one like this: 
`constructor(...args) {
  super(...args);
}
`
But I can be wrong, am I?

Reply:
That's true. I am a bit to used to languages, where you must call higher constructor. My bad.